pyanaconda/modules/storage/partitioning/interactive/utils.py

```diff
@@ -1090,6 +1090,7 @@ def _destroy_device(storage, device):
     # is preexisting, reinitialize the disk.
```

Comment:
This is unrelated, but I just can't figure out how we know that it is 'the last' partition as the comment says.

Reply:
That magic is done by `config.can_initialize(storage, device.disk)` where we check if the disk is empty. That's not great, but it is one of the places in the installer I am afraid to touch as long as it works.